docs/examples/import-https-url-definition/rabbitmq.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: import-https-url-definitions 
+spec:
+  replicas: 1
+  resources:
+    requests:
+      cpu: 2
+      memory: 8Gi
+    limits:
+      cpu: 2
+      memory: 8Gi
```

Comment:
Are the resource request and limit required for this example to work? If not, lets remove it to keep the example simple and just about the importing definition itself.